Rakefile

```diff
@@ -19,4 +19,4 @@ Rake::TestTask.new(:test_unit) do |t|
 end
 
 Rake::Task[:default].clear if Rake::Task.task_defined?(:default)
-task default: %i[lint test_unit jasmine:ci pact:verify]
+task default: %i[test_unit jasmine:ci pact:verify lint]
```

Comment:
I always wondered why this varies between apps.
I think I prefer it this way, otherwise you have to switch to a more specific bundle exec test command if you've dropped `binding.pry` in to explore some tests.